webshare/share-simple-manual.html

```diff
@@ -14,7 +14,14 @@
         const url = 'https://www.example.com/some/path?some_query#some_fragment';
         setupManualShareTest(false, 'the title', 'the message', url);
         callWhenButtonClicked(() => navigator.share(
-              {title: 'the title', text: 'the message', url: url}));
+              {title: 'the title', text: 'the message', url: url})).then(
+              result => {
+                // Check that promise resolved with undefined. This is guarded
```

Comment:
Even with the comment I don't understand why the conditional helps. if `result === undefined`, then `assert_equals(result, undefined)` won't do anything visibly. I think this test is already relying on `Tests.prototype.set_file_is_test` in the end, so there is a test to fail even though it's implicit, right?